bin/whisper-fetch.py

```diff
@@ -64,14 +64,12 @@
 except (whisper.WhisperException, IOError) as exc:
   raise SystemExit('[ERROR] %s' % str(exc))
 
-if options.drop:
-  fcn = _DROP_FUNCTIONS.get(options.drop)
-  values = [x for x in values if fcn(x)]
+keep = _DROP_FUNCTIONS.get(options.drop, lambda: True)
```

Comment:
Maybe it's better to hide it behind: `if options.drop:`?
I'm worrying about performance here and below.

Reply:
Done in 8afd813